components/o-forms/src/js/forms.js

```diff
@@ -35,6 +35,21 @@ class Forms {
 			});
 		}
 	}
+	get formInputs() {
+		if (this.form) {
+			const formElements = Array.from(this.form.elements);
+			const inputsToRemove = this._formInputsCache.filter(input => !formElements.includes(input.input));
+			const elementsToAdd = formElements.filter(element => !this._formInputsCache.find((input) => input.input === element));
+			inputsToRemove.forEach(input => input.destroy());
+			this._formInputsCache = [
+				...this._formInputsCache.filter(input => !inputsToRemove.includes(input)),
+				...elementsToAdd.map(element => new Input(element))
+			];
+			return this._formInputsCache;
+		}
+		return null;
```

Comment:
Previously `formInputs` was always an array, but now may be `null`. As a public property a user may rely on this being an array, for example calling `forEach`, and so this could break their code – we don't do a good job of documenting types in Origami components and should look to improve that 😬 

Instead we could return an empty array to indicate no form fields have been found.
```suggestion
		return [];
```